test/core_dom/resource_url_resolver_spec.dart

```diff
@@ -242,7 +242,7 @@ void main() {
     _run_resolver(useRelativeUrls: false);
   });
 
-  describe('url_resolver uri rewrite', () {
+  group('url_resolver', () {
```

Comment:
Sorry jumping between unittest and jasmine syntax too much. I'll change it back to describe.